flink-autoscaler/src/main/java/org/apache/flink/autoscaler/utils/AutoScalerUtils.java

```diff
@@ -94,4 +99,108 @@ public static boolean excludeVerticesFromScaling(
         conf.set(AutoScalerOptions.VERTEX_EXCLUDE_IDS, new ArrayList<>(excludedIds));
         return anyAdded;
     }
+
+    /** Quartz doesn't have the invertTimeRange flag so rewrite this method. */
+    static boolean isTimeIncluded(CronCalendar cron, long timeInMillis) {
```

Comment:
Can we move these methods to a new class, e.g. `DateUtils` or `CronUtils`?

Reply:
Removed to `CalendarUtils` and corresponding tests removed to `CalendarUtilsTest`